### DISCUSSION TITLE
After scanning the code and logging in, the associated device on WhatsApp displays in Chinese: "Small and Medium Business Agent". Can anyone tell me what's going on and how to change it? Thank you very much.

### DISCUSSION THREAD

**Discussion by super-zzc** [tulir/whatsmeow#687] [WhatsApp protocol Q&A] [answered]
After scanning the code and logging in, the associated device on WhatsApp displays in Chinese: "Small and Medium Business Agent". Can anyone tell me what's going on and how to change it? Thank you very much.
![7621730524413_ pic](https://github.com/user-attachments/assets/2cc36715-7660-4163-8965-842031c8ec10)

**Comment by ghost** [comment, marked as answer]
That is device/client name.

https://github.com/tulir/whatsmeow/discussions/469#discussioncomment-7133463

**Reply by super-zzc** [reply]
Thank you very much for your help, the issue has been resolved.
